libstdc++/README.md

```diff
@@ -0,0 +1,81 @@
+# Tests for libstdc++
+
+## Prerequisites
+
+Tests demand `msgfmt` and `dejagnu` to be installed. You can do it by typing:
```

Comment:
I'd opt for adding a link to DejaGnu documentation:
https://www.gnu.org/software/dejagnu/